constraintFunctions.f90

```diff
@@ -149,6 +149,9 @@ subroutine getEnvVarsAtT( t )
     character(len=maxEnvVarNameLength) :: this_env_var_name, orderedEnvVarNames(size( envVarNames ))
     logical :: pressure_set, rh_set, temp_set
 
+    ! TODO, is this the best place to call calcTheta?
```

Comment:
This looks like a great place - the only functions that use `theta` are `calcDec()` and `calcZenith()`, and they are only called below.